src-java/testing/test-library/src/main/java/org/openkilda/testing/config/KafkaTopicNameConfig.java

```diff
@@ -37,11 +37,37 @@
  * components via {@link org.springframework.context.annotation.Import} annotation.
  */
 @Configuration
-@EnableAutoConfiguration(exclude = {ErrorMvcAutoConfiguration.class})
-@ConfigurationType(name = "kafkaTopicsConfig", value = KafkaTopicsConfig.class)
-@ConfigurationType(EnvironmentConfig.class)
 public class KafkaTopicNameConfig {
 
+    private KafkaTopicsConfig kafkaTopicsConfig;
+    private EnvironmentConfig environmentConfig;
+
+    @Bean
```

Comment:
@IvanChupin maybe we still need to think about the profiles or it's the task for the future? What do you think?